tests/test_core.py

```diff
@@ -827,3 +827,34 @@ def test_jagged_list_to_array_empty():
     left = np.array([[]], dtype="float64")
     right = core._jagged_list_to_array(arr, fill_value=np.nan, dtype="float64")
     npt.assert_array_equal(left, right)
+
+
+def naive_get_mask_slices(mask):
```

Comment:
Please move this naive implementation to the top of the test file. See other test files to see the location of the naive implementation

Reply:
> Please move this naive implementation to the top of the test file. See other test files to see the location of the naive implementation

I moved it to the TOP of the `test_core.py` file. 
FYI: Just in case you missed it, NOT all naive implementations are at the TOP. there are some `test` functions between the `naive` implementations.

Reply:
Okay, would you mind creating a new issue to track this? I think it's best when the naive implementations are at the top of the file and then followed by the tests below it